### PR TITLE
test: auth repo keycloak

### LINKED ISSUE #190: Implement tests for auth folder in storage

### PR DESCRIPTION
**Pull request by choffmann**
Close #190

### CODE REVIEW

File: internal/storage/auth/keycloak/testutils/utils.go [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+	}
+}
+
+func (s *KeycloakTestSuite) TestUserToCreateFunc() []*entities.User {
+	n := 20
+	users := make([]*entities.User, n)
+	for i := 0; i < n; i++ {
+		users[i] = &entities.User{
+			Username:    fmt.Sprintf("test%d", i),
+			FirstName:   "Toni",
```
Comment (doriengr) [inline internal/storage/auth/keycloak/testutils/utils.go:74]: Toni Tester is baaaaack 🔥

File: internal/storage/auth/keycloak/token_test.go [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+		// then
+		assert.NotNil(t, got)
+		assert.NotNil(t, got.Active)
+		assert.False(t, *got.Active)
+		assert.NoError(t, err)
+	})
+}
+
+func TestKeyCloakRepo_GetAccessTokenFromClientCode(t *testing.T) {
+	t.Skip("don't know how to test this yet, need to recreate the login flow. Will come back to this later")
```
Comment (doriengr) [inline internal/storage/auth/keycloak/token_test.go:65]: Let's add a TODO and an issue? Otherwise we'll forget it

**Review by doriengr — CHANGES_REQUESTED** [review]
🔥

File: internal/storage/auth/keycloak/token_test.go (hunk 2 above)
Comment (choffmann) [reply internal/storage/auth/keycloak/token_test.go:66]: #225

**Pull request merged**